io.github.rinigus.OSMScoutServer.json

```diff
@@ -0,0 +1,212 @@
+{
+    "app-id": "io.github.rinigus.OSMScoutServer",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "osmscout-server",
+    "finish-args": [
+        "--filesystem=host",
+        "--device=dri",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network"
+    ],
+    "rename-icon": "osmscout-server",
+    "cleanup": ["/include", "/lib/mapnik", "/lib/pkgconfig",
```

Comment:
I would suggest moving the module specific cleanups to their specific module to improve readablity. Also put a new entry on a new line.

Reply:
All put on separate lines, for keeping it concise, I left it as a global cleanup